thoth/common/openshift.py

```diff
@@ -472,6 +474,19 @@ def get_jobs(self, label_selector: str, namespace: str = None) -> dict:
 
         return response
 
+    def _get_template(self, _label_selector: str) -> dict:
+        """Get template from infra namespace, use label_selector to identify which template to get."""
+        response = self.ocp_client.resources.get(
+            api_version="template.openshift.io/v1", kind="Template"
+        ).get(namespace=self.infra_namespace, label_selector=_label_selector)
+        _LOGGER.debug(
+            "OpenShift response for getting provenance-checker template: %r",
```

Comment:
This one is very specific for just provenance-checker, but the function used for other templates as well. What about logging just label selector which should be descriptive enough?